README.md

```diff
@@ -1,111 +1,87 @@
 
-- [**Join our community for support, to give us feedback, or chat with us**](https://www.inngest.com/discord).
-- [Docs](https://www.inngest.com/docs)
-- [Read more about our vision and why this project exists](https://www.inngest.com/blog/open-source-event-driven-queue)
+[Read more about our vision and why this project exists](https://www.inngest.com/blog/open-source-event-driven-queue)
 
-<br />
 <br />
 
-## Installing Inngest
+## Quick Start
 
-Quick start - this downloads inngest for your os+arch into ./inngest:
+1. Install the Inngest CLI to get started:
 
-```
+```shell
 curl -sfL https://cli.inngest.com/install.sh | sh \
   && sudo mv ./inngest /usr/local/bin/inngest
 ```
 
-**Manually:** by downloading a [pre-compiled binary](https://github.com/inngest/inngest-cli/releases) and placing the binary in your path.
-
-<br />
-
-## Trying Inngest
-
-Once you've installed the CLI, you can run `inngest init` to build new functions, `inngest run` to test them, and `inngest dev` to run the dev server.
-
-- [Quick start docs](https://www.inngest.com/docs/quick-start)
+2.  Create a new function. It will prompt you to select a programming language and what event will trigger your function. Optionally use the `--trigger` flag to specify the event name:
 
-<br />
-
-## Example usage
-
-**Send events** to trigger background jobs anywhere in your own app:
-
-```tsx
-const host = "http://..."; // The inngest server.
-
-// Send events via a single HTTP POST using any language.  Here's JS:
-await fetch(host, {
-  method: "POST",
-  headers: { "content-type": "application/json" },
-  body: JSON.stringify({
-    name: "user/signed.up",
-    data: { id: user.id },
-    user: { external_id: user.id },
-  }),
-});
+```shell
+inngest init --trigger test/my.event
 ```
 
-**Run a function** whenever `user/signed.up` is received (created via `inngest init`):
+3. Run your new hello world function with dummy data:
 
-```tsx
-import type { EventTriggers } from "./types";
+```shell
+inngest run
+```
 
-export async function run({ event }: { event: EventTriggers }) {
-  const { id } = event.data; // The event data sent above.
+4. Run the Inngest DevServer. This starts a local "Event API" which can receive events. When events are received, functions with matching triggers will automatically be run. Optionally use the `-p` flag to specify the sport for the Event API.
 
-  // You can do anything here:  add the user to Stripe, Zendesk, LaunchDarkly,
-  // send welcome emails, start a drip campaign - whatever you'd do
-  // in the background.
-  return { status: 200 };
-}
+```shell
+inngest dev -p 9999
 ```
 
-```json
-{
-  "name": "Post-signup",
-  "id": "abstract-nitty-a14c1d",
-  "triggers": [{ "event": "user/signed.up" }]
-}
+5. Send events to the DevServer. Send right from your application using HTTP + JSON or simply, as a curl with a dummy key of `KEY`.
+
+```shell
+curl -X POST --data '{"name":"test/my.event","data":{"test":true}}' http://127.0.0.1:9999/e/KEY
```

Comment:
Should we change this to `demo/event.sent` to start moving all of the onboarding events to that namespace?